src/app/modules/admin/userlist/userlist.component.ts

```diff
@@ -61,21 +63,11 @@ export class UserlistComponent implements OnInit {
    @type {boolean}
    */
   public popupmenu: boolean = false;
```

Comment:
Not a big issue but would be nice if the properties would be before the constructor